lms/static/sass/views/_program-details.scss

```diff
@@ -655,6 +617,62 @@
   }
 }
 
+
+// CSS used in both the main program detail and the program detail sidebar
+.btn-view-detail {
+  background: palette(primary, dark);
+  border-color: palette(primary, dark);
+  height: 37px;
+  width: auto;
+  border-radius: 0;
+  text-align: center;
+  font-size: 0.9375em;
+  margin-top: 5px;
+
+  /* IE11 CSS styles */
+  @media (min-width: $bp-screen-md) and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    @include float(right);
+  }
+
+  @media (min-width: $bp-screen-lg) {
+    margin-top: 0;
+  }
+}
+
+.detail-heading {
+  font-family: $font-family-sans-serif;
+  color: palette(primary, dark);
+  font-size: 0.9375em;
+  line-height: normal;
+  padding-bottom: 5px;
+  border-bottom: 3px solid $divider-color;
+  margin: {
```

Comment:
is this our best practice now? If not I could collapse to short-hand margin rule on 1 line